ci/do_ci.sh

```diff
@@ -412,6 +412,8 @@ elif [[ "$CI_TARGET" == "docs" ]]; then
   # Validate dependency relationships between core/extensions and external deps.
   tools/dependency/validate_test.py
   tools/dependency/validate.py
+  # Validate the CVE scanner works. TODO(htuch): create a dedicated tools CI target.
+  python3.8 tools/dependency/cve_scan_test.py
```

Comment:
Does this need the `python3.8 `?

Reply:
Yeah, the datetime methods we use are more recent.